kentikapi/client.go

```diff
@@ -78,26 +75,67 @@ type Config struct {
 	// A Timeout of zero means no timeout.
 	// Default: 100 seconds.
 	Timeout *time.Duration
-	// DisableTLS disables TLS for client connections.
-	// It has effect only on gRPC services: CloudExportAdmin, SyntheticsAdmin and SyntheticsData.
-	DisableTLS bool
 }
 
 type RetryConfig = httputil.RetryConfig
 
+type ConfigOption func(*Config)
```

Comment:
```suggestion
type ClientOption func(*Config)
```
It's an option to the client.

Reply:
Done